usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -392,6 +394,21 @@ def get(self, request, recipient_id):
         return Response(result)
 
 
+@method_decorator(deprecated, name="get")
+class RecipientOverViewDuns(RecipientOverView):
+    """
+    Deprecated: This endpoint returns a high-level overview of a specific recipient, given its id.
```

Comment:
Can you copy the pattern that Todd used here for showing deprecated on the API UI: https://github.com/fedspendingtransparency/usaspending-api/blob/master/usaspending_api/search/v2/views/spending_by_category_views/spending_by_recipient.py#L101